values/gatekeeper-operator/gatekeeper-operator-raw.gotmpl

```diff
@@ -0,0 +1,6 @@
+{{- $v := .Environment.Values }}
+{{- $constraints := (exec "bash" (list "-c" "find  ./constraint-templates -name '*.yaml'")) | splitList "\n" }}
+resources:
+{{- range $constraint := $constraints }}{{ if ne $constraint "" }}
```

Comment:
why the defensive `if ne $constraint ""`? is the above exec not good enough? then fix it